### PR TITLE
feat: publish @podman-desktop/webview-api to npmjs

### LINKED ISSUE #6380: publish webview api to @podman-desktop/webview-api
### Is your enhancement related to a problem? Please describe

I see that some extensions are declaring a .d.ts file 

with

```
declare global {
  export interface PodmanDesktopApi {
    getState: () => any;
    postMessage: (msg: any) => void;
    setState: (newState: any) => void;
  }

  function acquirePodmanDesktopApi(): PodmanDesktopApi;
}

export { PodmanDesktopApi };
````

we should try to look if we can publish the declaration of this API so extensions don't need to declare it

### Describe the solution you'd like

publish the declaration if possible

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
it allows to get typechecking for webviews

### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/6380

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/webview-api/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "node16",
+    "baseUrl": ".",
+    "types": ["node"],
```
Comment (axel7083) [inline packages/webview-api/tsconfig.json]: node ? might not be required

Comment (benoitf) [reply packages/webview-api/tsconfig.json]: yes 👍

**Review by axel7083 — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]

File: packages/preload-webview/src/webview-preload.spec.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 import { afterEach } from 'node:test';
 
+import type { WebviewApi } from '@podman-desktop/webview-api';
```
Comment (dgolovin) [inline packages/preload-webview/src/webview-preload.spec.ts:23]: Webview is UI, why not to have it as a part of packages/ui and have it in code like

```
import type { WebviewApi } from '@podman-desktop/ui-svelte';
```
?

Comment (benoitf) [reply packages/preload-webview/src/webview-preload.spec.ts:23]: you don't need the UI package to work with Webviews

**Pull request merged**